forms/aws/lambda/reliability/lib/dataLayer.js

```diff
@@ -93,23 +93,21 @@ function extractFileInputResponses(submission) {
     return [...fileInputElements, ...dynamicRowElementsIncludingFileInputComponents];
 }
 
-function extractFormData(submission) {
+function extractFormData(submission, language) {
   const formResponses = submission.responses;
   const formOrigin = submission.form;
   const dataCollector = [];
   formOrigin.layout.map((qID) => {
     const question = formOrigin.elements.find((element) => element.id === qID);
     if (question) {
-      handleType(question, formResponses[question.id], dataCollector);
+      handleType(question, formResponses[question.id], language, dataCollector);
     }
   });
   return dataCollector;
 }
 
-function handleType(question, response, collector) {
-  // Add i18n here later on?
-  // Do we detect lang submission or output with mixed lang?
-  const qTitle = question.properties.titleEn;
+function handleType(question, response, language, collector) {
+  const qTitle = language === "fr"? question.properties.titleFr : question.properties.titleEn;
```

Comment:
Although it should still work I'm surprised that prettier didn't pickup on the missing ' ' between "fr" and "?"